code/components/net-base/src/NetPeerBase.cpp

```diff
@@ -29,7 +29,7 @@ void PeerBase::ProcessPacket(const std::vector<uint8_t>& buffer)
 int PeerBase::ReadCompressedType(Buffer& buffer)
 {
 	uint8_t lead = buffer.Read<uint8_t>();
-	int result;
+	int result = -1;

```

Comment:
Not sure if this code was ever used... it might have been some early test of a custom net wrapper. Dead code, yay!